STACpopulator/implementations/CMIP6_UofT/add_CMIP6.py

```diff
@@ -123,13 +123,14 @@ def runner(ns: argparse.Namespace) -> Optional[int] | NoReturn:
             ns.stac_host, data_loader, update=ns.update, session=session, config_file=ns.config, log_debug=ns.debug
         )
         c.ingest()
+    return 0
 
 
-def main(*args: str) -> Optional[int]:
-    parser = make_parser()
+def main(*args: str) -> int:
+    parser = argparse.ArgumentParser()
```

Comment:
If one calls `python STACpopulator/implementations/CMIP6_UofT/add_CMIP6.py`, this will now fail. 
The previous `make_parser` approach allowed to do either approach. Is it only missing a `add_parser_args` call to accomplish the same?

Reply:
Yup it's missing. I'll add it in. 

Personally I don't love that there are multiple interfaces for running this script and I really think that we should discourage running these files directly. But that's a discussion for another time.

Reply:
(this is the reason that I didn't include more tests for running these scripts directly from the command line)